perf/perf.c

```diff
@@ -0,0 +1,79 @@
+/************************************************************************/
+/*         *   The Coq Proof Assistant / The Coq Development Team       */
+/*  v      *         Copyright INRIA, CNRS and contributors             */
+/* <O___,, * (see version control and CREDITS file for authors & dates) */
+/*   \VV/  **************************************************************/
+/*    //   *    This file is distributed under the terms of the         */
+/*         *     GNU Lesser General Public License Version 2.1          */
+/*         *     (see LICENSE file for the text of the license)         */
+/************************************************************************/
+
+// Copyright (C) BedRock Systems Inc. 2023 Rodolphe Lepigre
```

Comment:
Some time ago we gave up asking for signing a contribution agreement, but I think we don't want this kind of line either. Others from BedRock did contribute before and are acknowledged in the changes file, but they did
not change the files headers.

CC @ybertot @maximedenes

Reply:
Sure, I grepped to see what was the usual way to do this, and found some files with similar lines (including `ssreflect`). But then again, it is not like the source code is in any way consistent through the repo. In any case, I was not asked to put anything like this in files, so I'm happy to remove.